### PR TITLE
track branch along with lines-of-code

### PR DESCRIPTION
**Pull request by apucacao**
With this change, every metric will be tagged with the name of the branch that received the commit. (Once we have that, we could see how much "impact" a PR hold.)

I went down the rabbit hole fixing edge-cases, removing duplication, and streamlining how we collect the info we need, so that we can start reporting on these metrics in PR as well.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -51,7 +52,7 @@ async function submitToDataDog(
             metric: datadogMetric,
             type: seriesType,
             points: [[timestamp, dataPoint]],
-            tags: [`author:${author}`],
+            tags: [`author:${author}`, `branch:${branch}`],
```
Comment (apucacao) [inline src/index.ts:55]: Ignoring the cleanup-related changes, it all boils down to this.

File: src/index.ts [hunk 2]
```diff
@@ -62,53 +63,112 @@ async function submitToDataDog(
...
-    const otherFiles = response.files ? response.files.filter((f: { filename?: string }) => f.filename) : [];
+    const otherFiles = commit.files ? commit.files.filter((f: { filename?: string }) => f.filename) : [];
     const count = findFileCountOfJSConversionsToTS(renamedFiles);
     const otherCount = findFileCountOfJSConversionsToTSForAllFiles(otherFiles);
     const totalCount = count + otherCount;
 
-    //do not report 0 counts
+    // do not report 0 counts
     if (totalCount === 0) {
+      core.info('Ignoring commit with no file changes');
```
Comment (traciklopez) [inline src/index.ts:152]: We're ignoring commits with no JS or TS/X file changes.

File: example/moduleE.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+console.log(123);
```
Comment (traciklopez) [inline example/moduleE.ts:1]: what is this?

**Review by traciklopez — APPROVED** [review]

File: example/moduleE.ts (hunk 3 above)
Comment (apucacao) [reply example/moduleE.ts:1]: i wanted to test that we handled the "push to a branch" github action flow properly, so I,
- added a js file
- created a PR to migrate it
- merged it into this branch

(That replicates what happens when we merge PRs into master.)

File: src/index.ts (hunk 2 above)
Comment (apucacao) [reply src/index.ts]: updated, thanks

**Pull request merged**